DasharoModulePkg/UserAuthenticationDxe/UserAuthenticationDxeStrings.uni

```diff
@@ -19,8 +19,8 @@
                                        #language fr-FR "Password Management"
 #string STR_ADMIN_PASSWORD_PROMPT      #language en-US "Change Admin Password"
                                        #language fr-FR "Change Admin Password"
-#string STR_ADMIN_PASSWORD_HELP        #language en-US "Input old admin password if it was set, then you can change the password to a new one. After the change action, you may need input the new password when you enter UI. The new password must be between 8 and 32 chars include lowercase, uppercase alphabetic, number, and symbol. Input an empty password can clean old admin password, then no need input password to enter UI."
-                                       #language fr-FR "Input old admin password if it was set, then you can change the password to a new one. After the change action, you may need input the new password when you enter UI. The new password must be between 8 and 32 chars include lowercase, uppercase alphabetic, number, and symbol. Input an empty password can clean old admin password, then no need input password to enter UI."
+#string STR_ADMIN_PASSWORD_HELP        #language en-US "Input old admin password if it was set, then you can change the password to a new one. After the change action, you may need input the new password when you enter UI. The new password must be between 8 and 32 chars. Input an empty password can clean old admin password, then no need input password to enter UI."

```

Comment:
32 might be a bit too constrained for a passphrase, can we have e.g. 64?

Reply:
I think I have linked in the issue this - https://cheatsheetseries.owasp.org/cheatsheets/Authentication_Cheat_Sheet.html#implement-proper-password-strength-controls 

I think we can refer to that and use these guidelines, unless we have better ones

Reply:
> 32 might be a bit too constrained for a passphrase, can we have e.g. 64?
That's what I also thought. https://cheatsheetseries.owasp.org/cheatsheets/Authentication_Cheat_Sheet.html#implement-proper-password-strength-controls Exactly, here, as well as in the NIST Special Publication 800-63B they reference it is said to allow at least 64 characters. I will do it.